src/connectors/search-box/__tests__/__snapshots__/connectSearchBox-test.js.snap

```diff
@@ -1,37 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`connectSearchBox routing getWidgetSearchParameters should add the refinements according to the UI state provided 1`] = `
-SearchParameters {
-  "disjunctiveFacets": Array [],
-  "disjunctiveFacetsRefinements": Object {},
-  "facets": Array [],
-  "facetsExcludes": Object {},
-  "facetsRefinements": Object {},
-  "hierarchicalFacets": Array [],
-  "hierarchicalFacetsRefinements": Object {},
-  "index": "",
-  "numericRefinements": Object {},
-  "query": "some search",
-  "tagRefinements": Array [],
-}
-`;
-
-exports[`connectSearchBox routing getWidgetSearchParameters should enforce the default value if no value is the ui state 1`] = `
-SearchParameters {
-  "disjunctiveFacets": Array [],
-  "disjunctiveFacetsRefinements": Object {},
-  "facets": Array [],
-  "facetsExcludes": Object {},
-  "facetsRefinements": Object {},
-  "hierarchicalFacets": Array [],
-  "hierarchicalFacetsRefinements": Object {},
-  "index": "",
-  "numericRefinements": Object {},
-  "tagRefinements": Array [],
-}
-`;
-
-exports[`connectSearchBox routing getWidgetState should add an entry equal to the refinement 1`] = `
+exports[`connectSearchBox getWidgetState should add an entry equal to the refinement 1`] = `
 Object {
   "query": "some query",
```

Comment:
let's remove this snapshot here as well

Reply:
I didn't to avoid to touch too much code. It's not related to this PR at all.